imagebuildah/stage_executor.go

```diff
@@ -2020,14 +2020,15 @@ func (s *StageExecutor) commit(ctx context.Context, createdBy string, emptyLayer
 		s.builder.SetLabel(k, v)
 	}
 	if s.executor.commonBuildOptions.IdentityLabel == types.OptionalBoolUndefined || s.executor.commonBuildOptions.IdentityLabel == types.OptionalBoolTrue {
-		s.builder.SetLabel(buildah.BuilderIdentityAnnotation, define.Version)
-	}
-	for _, labelSpec := range s.executor.labels {
-		label := strings.SplitN(labelSpec, "=", 2)
-		if len(label) > 1 {
-			s.builder.SetLabel(label[0], label[1])
-		} else {
-			s.builder.SetLabel(label[0], "")
+		didUserSetIndentityLabel := false
```

Comment:
nit, I'd drop the 'did'
```suggestion
		userSetIndentityLabel := false
```

Reply:
removed this commit